scheduler/src/main/java/io/camunda/zeebe/scheduler/health/HealthMonitor.java

```diff
@@ -5,7 +5,7 @@
  * Licensed under the Zeebe Community License 1.1. You may not use this file
  * except in compliance with the Zeebe Community License 1.1.
  */
-package io.camunda.zeebe.util.health;
+package io.camunda.zeebe.scheduler.health;
```

Comment:
:wrench: Here I'm unsure since these all are interface I could also imaging that this is an own module, unrelated to scheduler :shrug:

Reply:
True, everything in `health` apart from `CriticalComponentsHealthMonitor` could stay in `zeebe-util` I suppose

Reply:
I've added one more commit which moves most of the classes in health and error back to the utils module :+1:

Reply:
Thanks :)